docs/source/pages/community/community.md

```diff
@@ -0,0 +1,30 @@
+# Community
+
+
+Contributions to **datashuttle** are strongly encouraged. If you would like to submit
+a bug fix or implement a new feature, please feel free to do so! We are a
+friendly bunch and happy to answer any questions before,
+during or after a contribution. See the [Contributing Guide](contributing.md) for more details.
+
+We are generally very keen to hear feedback on what works and doesn't work
+in **datashuttle** and
+[**NeuroBlueprint**](https://neuroblueprint.neuroinformatics.dev).
+Please get in contact with any thoughts on how these tools can be
+improved by opening a
+[GitHub Issue](https://github.com/neuroinformatics-unit/datashuttle/issues)
+or by joining our
+[Zulip Chat](https://neuroinformatics.zulipchat.com/#narrow/stream/405999-DataShuttle).
+
+To see what we have planned for the future of **NeuroBlueprint** and **datashuttle**, visit
+our [Guiding Principles](guiding-principles.md) and [Roadmap](roadmap.md).
+
+
+```{toctree}
+:maxdepth: 2
+:caption: index
+:hidden:
```

Comment:
I wouldn't hide this. It would be nice for readers to be able to navigate directly, without parsing prose.

Reply:
Good point, I added the 'primary sidebar' back to the Community section:

![image](https://github.com/neuroinformatics-unit/datashuttle/assets/55797454/4f73d1ac-bcbd-46fa-b1be-58d4ca294651)